x-pack/plugins/actions/server/authorization/actions_authorization.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import Boom from 'boom';
+import { KibanaRequest } from 'src/core/server';
+import { SecurityPluginSetup } from '../../../security/server';
+import { ActionsAuthorizationAuditLogger } from './audit_logger';
+import { ACTION_SAVED_OBJECT_TYPE, ACTION_TASK_PARAMS_SAVED_OBJECT_TYPE } from '../saved_objects';
+
+export interface ConstructorOptions {
+  request: KibanaRequest;
+  auditLogger: ActionsAuthorizationAuditLogger;
+  authorization?: SecurityPluginSetup['authz'];
+}
+
+const operationAlias: Record<
+  string,
+  (authorization: SecurityPluginSetup['authz']) => string | string[]
+> = {
+  execute: (authorization) => [
+    authorization.actions.savedObject.get(ACTION_SAVED_OBJECT_TYPE, 'get'),
+    authorization.actions.savedObject.get(ACTION_TASK_PARAMS_SAVED_OBJECT_TYPE, 'create'),
+  ],
+  list: (authorization) => authorization.actions.savedObject.get(ACTION_SAVED_OBJECT_TYPE, 'get'),
+};
```

Comment:
As we're postponing the Solution specific actions RBAC to a follow up issue, we can rely on Saved Objects as a backing for the feature based RBAC